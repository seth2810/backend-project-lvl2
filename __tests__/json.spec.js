import genDiff from '../src/index.js';

describe('json', () => {
  describe('flat', () => {
    it('should return correct diff', () => {
      const result = genDiff(resolveFixturePath('json/flat/before.json'), resolveFixturePath('json/flat/after.json'));
      expect(result).toMatchSnapshot();
    });
  });
});
