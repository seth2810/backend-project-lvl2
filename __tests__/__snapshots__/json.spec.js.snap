// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`json flat should return correct diff 1`] = `
"{
    host: hexlet.io
  + timeout: 20
  - timeout: 50
  - proxy: 123.234.53.22
  - follow: false
  + verbose: true
}"
`;
